packages/site/pages/_app.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import FeatureFlags from '@pluralsight/ps-design-system-featureflags'
+
+const App = ({ Component, pageProps }) => (
+  <FeatureFlags flags={{ psds2020Colors: true }}>
```

Comment:
This makes it much more "apply new colors to everything", which is nice if we're optimizing for blank flag changing.  When I put it in the examples code last, I was thinking in terms of putting a flag toggle on those particular examples that would respond to it.  But, I suppose this doesn't preclude that someday, as long as they nest.